content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -159,7 +159,6 @@ Feature | Default | Alpha | Beta | GA
 :-------|:--------|:------|:-----|:----
 `PublicKeysECDSA` | `false` | 1.19 | - | -
 `RootlessControlPlane` | `false` | 1.22 | - | -
-`UnversionedKubeletConfigMap` | `true` | 1.22 | 1.23 | 1.25
```

Comment:
i think sig docs wanted to keep track of okd fgs

Reply:
good idea.